tests/scenario_tests/test_middleware.py

```diff
@@ -87,6 +87,53 @@ def test_next_call(self):
         assert response.body == "acknowledged!"
         assert_auth_test_count(self, 1)
 
+    def test_decorator_next_call(self):
```

Comment:
Checking these test patterns should be helpful for knowing the purpose of the changes.